src/constraints/investment.jl

```diff
@@ -6,19 +6,17 @@ add_investment_constraints!(graph, Ai, Ase, Fi, assets_investment, assets_invest
 Adds the investment constraints for all asset types and transport flows to the model
 """
 
-function add_investment_constraints!(
-    graph,
-    sets,
-    assets_investment,
-    assets_investment_energy,
-    flows_investment,
-)
+function add_investment_constraints!(graph, sets, variables)
```

Comment:
Add "TODO: Differentiate the type of constraint creation (bound vs not bound)"
Add "TODO: Verify if it's possible to move the bound to the indices creation"

Reply:
Done